### DISCUSSION TITLE
Apply CPU Limit : High PL1&PL2 values display in HWinfo

### DISCUSSION THREAD

**Discussion by Kikimahe** [seerge/g-helper#1839] [Q&A] [answered]
Hi and Happy New year to Seerge and all lucky Ghelper Users !  

This is not a problem, it's just to understand how it works. :)

When I apply a CPU limit in Ghelper from Turbo mode (40W-70W), I observe that the PL1 and PL2 values are strangely very high in HWINFO:
- PL1 115W
- PL2 135W 

Theoretically my CPU obviously can't reach these values (90W-115W max).

Shouldn't the PL1 and PL2 values displayed in HWinfo be the same as those defined in Ghelper ?

![Sans titre-1](https://github.com/seerge/g-helper/assets/142881210/923d1834-d2a2-4bce-9b67-5409eaffc9bc)

[log.txt](https://github.com/seerge/g-helper/files/13817788/log.txt)

**Comment by seerge** [comment, marked as answer]
@Kikimahe G-Helper just sets those power limit values in special Asus endpoints. On Intel they don't affect PL1/PL2 values that  HWinfo would show, but rather limit real power CPU can draw. Most probably Asus just limits it somewhere on motherboard level. 

Easiest way to test if it works - is to run a CPU heavy benchmark (like Cinebench) and just monitor actual CPU power draw in HWinfo

**Comment by Kikimahe** [comment]
Thank you for your explanations Seerge.

Yes before this message, I did tests on different benchmark software and checked the CPU consumption. Everything is ok.

But I was just surprised that HWinfo immediately shows a value of 115W/135W only when I apply a limit in Ghelper. My CPU will never be able to reach these values :)

**Reply by seerge** [reply]
Most probably HWinfo gets PL1/PL2 from chipset driver in windows or something like that, and asus sets them differently

**Reply by Kikimahe** [reply]
I'm going to ask Martin the question on the Hwinfo forum. Maybe he can tell me more. ;-)
